Requirements.md

```diff
@@ -69,6 +70,42 @@ Game Marketplace:
 - A section of the platform where users can discover and playtest new games.
 - Option for developers to monetize games through in-game purchases, ads, or premium versions.
 
+### 2.4 Events
+
+General Meetings:
+
+- Meetings can be created by any logged-in user on the platform at any time.
+- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development.
```

Comment:
"- Users can submit meetings for evaluation, including workshops, lectures, or other activities related to game development."
I think it is mis placed. it should be on choice cast, right?